src/main/java/gov/usgs/owi/nldi/controllers/CharacteristicsController.java

```diff
@@ -75,4 +75,18 @@ public void getCharacteristicData(HttpServletRequest request, HttpServletRespons
 		}
 		logService.logRequestComplete(logId, response.getStatus());
 	}
+
+	@GetMapping(value="{featureSource}/{featureID}/basin")
+	public void getBasin(HttpServletRequest request, HttpServletResponse response,
+			@PathVariable(LookupDao.FEATURE_SOURCE) String featureSource,
+			@PathVariable(Parameters.FEATURE_ID) String featureID) throws IOException {
+		BigInteger logId = logService.logRequest(request);
+		try {
+			streamBasin(response, getComid(featureSource, featureID));
+		} catch (Throwable e) {
```

Comment:
The [javadocs](https://docs.oracle.com/javase/8/docs/api/java/lang/Error.html) contraindicate catching Errors:

> An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch.